google/ggrc-core
# GGRC-6279 If change GCA value via import, Control is not reverted to Unreviewed state

# Issue description
The ImportRowConverter  -> send_pre_commit_signals method starts before ImportRowConverter -> setup_secondary_objects. That is why the Reviewable -> _update_status_on_custom_attrs method does not see changes in the GCA, since the setting of GCA makes in the "setup_secondary_objects" method.

# Steps to test the changes

1. Go to the administration panel
2. Create one or more GCA (Global Custom Attribute) for Controls
3. Create a control
4. Mark reviewed
5. Export Control to Google sheet
6. Update any GCA (Global Custom Attribute) > Save
7. Import Control
8. Look at the review state: Control should be reverted to unreviewed state

# Solution description
The easiest way to solve this problem is to  set call of the ImportRowConverter  -> send_pre_commit_signals method once again after the ImportRowConverter -> setup_secondary_objects method before commit.

# Sanity checklist


# PR Review checklist
